src/main/java/com/powsybl/openloadflow/network/impl/LfGeneratorImpl.java

```diff
@@ -30,11 +30,17 @@ public final class LfGeneratorImpl extends AbstractLfGenerator {
 
     private double droop;
 
+    private LfNetworkParameters parameters;
+
+    private LfNetworkLoadingReport report;
+
     private LfGeneratorImpl(Generator generator, LfNetwork network, LfNetworkParameters parameters, LfNetworkLoadingReport report) {
         super(network, generator.getTargetP());
         this.generatorRef = new Ref<>(generator);
         participating = true;
         droop = DEFAULT_DROOP;
+        this.parameters = parameters;
+        this.report = report;
```

Comment:
These parameters are used, for now, to call the setVoltageControl method for the setTargetV update of the GeneratorAction. Are they better ways to do so? Should we re-think the way these things are passed?